feature-detects/css/vhunit.js

```diff
@@ -14,11 +14,9 @@
   }]
 }
 !*/
-define(['Modernizr', 'testStyles', 'computedStyle', 'roundedEquals'], function(Modernizr, testStyles, computedStyle, roundedEquals) {
-  testStyles('#modernizr { height: 50vh; }', function(elem) {
-    var height = parseInt(window.innerHeight / 2, 10);
+define(['Modernizr', 'testStyles', 'computedStyle'], function(Modernizr, testStyles, computedStyle) {
+  testStyles('#modernizr { height: 50vh; max-height: 10px; }', function(elem) {
     var compStyle = parseInt(computedStyle(elem, null, 'height'), 10);
-
-    Modernizr.addTest('cssvhunit', roundedEquals(compStyle, height));
+    Modernizr.addTest('cssvhunit', compStyle == 10);
```

Comment:
why == instead of === ?

Reply:
Both of them have type "Number", don't they? I can change to '===' not a problem at all. Should I do it?

Reply:
=== is in 99 % of the time preferable even if you think that the types are always the same...

Reply:
Fixed! :)